test/test_transforms_v2_refactored.py

```diff
@@ -1867,7 +1872,7 @@ def test_others_catch_all_and_none(self, make_input):
         # make sure "others" works as a catch-all and that None means no conversion
 
         sample, inpt_dtype, bbox_dtype, mask_dtype = self.make_inpt_with_bbox_and_mask(make_input)
-        out = transforms.ToDtype(dtype={tv_tensors.Mask: torch.int64, "others": None})(sample)
+        out = transforms.ToDtype(dtype={tv_tensors.DetectionMasks: torch.int64, "others": None})(sample)
```

Comment:
We need to fix this in our references as well.

Reply:
We should not need to change this?  Passing `tv_tensors.Mask: torch.int64` should still work, right?